utils/merkletrie/difftree.go

```diff
@@ -448,3 +454,18 @@ func diffDirs(changes *Changes, ii *doubleIter) error {
 
 	return nil
 }
+
+func isIgnoredNode(tree noder.Noder, path noder.Path) bool {
```

Comment:
```suggestion

// isIgnoredPath reports if the path is ignored in the tree.
func isIgnoredPath(tree noder.Noder, path noder.Path) bool {
	in, ok := tree.(*gitignore.MatchNoder)

	return ok && in.PathIgnored(path)
}
```
Maybe this is more intuitive?